hexa/plugins/connector_airflow/schema.py

```diff
@@ -262,11 +258,6 @@ def resolve_prepare_download_url(_, info, **kwargs):
                 bucket=bucket,
                 target_key=uri_path,
             )
-        elif uri_protocol == "gcs":
```

Comment:
Could you remove the if statement above since we only have one way to do it please ?